packages/framework/src/state/user.ts

```diff
@@ -20,3 +20,7 @@ export const useEnsembleUser = (): [
   const [user, setUser] = useAtom(userAtom);
   return [user, setUser];
 };
+
+export const useEnsembleHistory = (): History => {
```

Comment:
What is this for?

Reply:
That is only for testing, and forgot to remove it before commit